models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatModelFunctionCallingIT.java

```diff
@@ -124,6 +125,7 @@ public OllamaChatModel ollamaChat(OllamaApi ollamaApi) {
 			return OllamaChatModel.builder()
 				.withOllamaApi(ollamaApi)
 				.withDefaultOptions(OllamaOptions.create().withModel(MODEL).withTemperature(0.9))
+				.withRetryTemplate(RetryUtils.DEFAULT_RETRY_TEMPLATE)
```

Comment:
Is this needed? If I read correctly, `RetryUtils.DEFAULT_RETRY_TEMPLATE` is provided by default if no explicit `RestTemplate` object is provided, so there shouldn't be a need for adding this line.

Same question for all the other places where this line was added.